Source/Data/BPMFMappings.txt

```diff
@@ -119755,6 +119814,7 @@
 軟骨 ㄖㄨㄢˇ ㄍㄨˇ
 軟骨病 ㄖㄨㄢˇ ㄍㄨˇ ㄅㄧㄥˋ
 軟骨頭 ㄖㄨㄢˇ ㄍㄨˇ ㄊㄡˊ
+軟骨頭 ㄖㄨㄢˇ ㄍㄨˇ ㄊㄡ˙
```

Comment:
For consistency `/ㄖㄨㄢˇ ㄍㄨˊ ˙ㄊㄡ/` should probably also be here.

Reply:
fixed.